app/atoms/ui/_ellipsis-multiline.scss

```diff
@@ -0,0 +1,16 @@
+// ------------------------------------------------------------
+// Text Ellipsis Multiline
+// ------------------------------------------------------------
+
+@mixin text-ellipsis($font-size, $line-height, $lines-to-show) {
```

Comment:
For now we could put all mixins in the same folder in the seed project, eg app/atoms/mixins.
We can separate it out later, but if it's at least in the seed project it can be re-used.